web-ui/demo/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
I'm not sure I understand why we have this new demo section?

Reply:
I'm with Jesse on this. It seems like it's here to show how to use the DnD stuff. I think that would be better put into the Storybook stories (just make a .stories.jsx file for it...see: https://storybook.js.org/docs/react/writing-stories/introduction)